src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  width: 950px;
+  margin: 0 auto;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background: $white-color;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+  margin-right: 50px;
+  margin-top: 51px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    align-self: center;
+  }
+
+  &__title {
+    color: $main-color;
+    margin-bottom: 4px;
+  }
+
+  &__pcode {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    color: $main-color;
+    font-weight: 400;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-count {
+    font-weight: 400;
+    font-size: 10px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-style: normal;
+    font-weight: 400;
```

Comment:
it's initial value so you don't need to write this
![image](https://user-images.githubusercontent.com/91411692/223058027-46cf68f0-2cbe-4c59-9830-d171f6c07a39.png)
![image](https://user-images.githubusercontent.com/91411692/223058129-6697732b-6564-43d8-bee7-f8c0413bfe9b.png)

```suggestion
```